tests/integration.rs

```diff
@@ -120,6 +120,37 @@ fn invert() -> Result<()> {
   assert_output_eq(&["resize:1:1", "all"], "1")
 }
 
+#[test]
+fn save() -> Result<()> {
+  let path = "output.png";
+
+  assert_output_eq(
+    &["resize:4:4", &format!("save:{}", path), "top"],
```

Comment:
Since forgetting to transpose is easy, let's use a different width and height so that this test will fail if we don't do it right. Also, I think we should check that the image non-zero, and that it's different from the final output.

```suggestion
    &["resize:4:2", "top", "save:output.png", "all],
```